### DISCUSSION TITLE
Is there a function which can parse kv binary?

### DISCUSSION THREAD

**Discussion by adminerest** [ValvePython/steam#410] [Q&A] [answered]
Is there a function which can parse kv binary?
Something like this: https://github.com/SteamRE/SteamKit/discussions/1024
And this: https://github.com/DoctorMcKay/node-binarykvparser

**Comment by Gobot1234** [comment, marked as answer]
Yes, you should haVe the vdf module installed already, import that and use the binary_loads function
